### PR TITLE
[IOPID-525] : created ko page in case of expired magic link 

### PR DESCRIPTION
**Pull request by OraldoDoci**
## Short description

New KO page and small graphic fix.

## List of changes proposed in this pull request

- Created KO Page in case the magic link has expired
- integrated a graphic fix for spinner during validation

## How to test

Run application
To check the new page go to http://localhost:3000/expiredMagicLink  
To check the style fix trigger validation and verify there is no blank page after the spinner stops

### CODE REVIEW

File: src/app/[locale]/(pages)/expiredMagicLink/page.tsx [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+            <Typography
+              variant="body1"
+              py={0}
+              px={0}
+              color="textPrimary"
+              sx={{
+                textAlign: 'center',
+              }}
+            >
+              Per bloccare l’accesso a IO è necessario accedere al portale con le tue credenziali
```
Comment (Ladirico) [inline src/app/[locale]/(pages)/expiredMagicLink/page.tsx:50]: Why you don't use the translations? The key is "lockiolog" into "error"

File: src/app/[locale]/(pages)/profileAccessBlocked/page.tsx [hunk 2]
```diff
@@ -6,16 +6,16 @@ import { ButtonNaked } from '@pagopa/mui-italia';
...
+import { commonBackground } from '../../_utils/styles';
 
 const Profile = () => {
   const t = useTranslations('ioesco');
   const bgColor = 'background.paper';
   return (
     <Grid sx={commonBackground}>
       <Introduction
-        title={t('common.hello', { Nome: 'Mario' })}
+        title={t('common.hello', { nome: 'Mario' })}
```
Comment (Ladirico) [inline src/app/[locale]/(pages)/profileAccessBlocked/page.tsx:18]: why you don't use the name from the token? (i have notice this issue in some pages)

File: src/app/[locale]/(pages)/expiredMagicLink/page.tsx [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+            <Typography
+              variant="h4"
+              py={0}
+              px={0}
+              color="textPrimary"
+              sx={{
+                textAlign: 'center',
+              }}
+            >
+              Il link è scaduto
```
Comment (Ladirico) [inline src/app/[locale]/(pages)/expiredMagicLink/page.tsx:37]: Why you don't use the translations? The key is "linkexpired" into "error"

**Review by Ladirico — CHANGES_REQUESTED** [review]

File: src/app/[locale]/(pages)/profileAccessBlocked/page.tsx (hunk 2 above)
Comment (OraldoDoci) [reply src/app/[locale]/(pages)/profileAccessBlocked/page.tsx:18]: The name variable will not be read from the session storage but will be returned from a api call that will be integrated in the future. For the moment we will leave it in the code until the new API is created and integrated.

File: src/app/[locale]/(pages)/expiredMagicLink/page.tsx (hunk 1 above)
Comment (OraldoDoci) [reply src/app/[locale]/(pages)/expiredMagicLink/page.tsx]: added missing translation

**Review by Ladirico — APPROVED** [review]

**Pull request merged**